File: src/common/actions.js
const telegram = require('./telegram');

const { TELEGRAM_INTERFACE_NAME } = process.env;

module.exports.replyMessage = (interface, message) => {
  if (interface.name == TELEGRAM_INTERFACE_NAME) {
    return telegram.sendMessage(interface.chatId, message);
  }

  throw new Error('Unknown interface name');
};

module.exports.replyPhoto = (interface, photoPath) => {
  if (interface.name == TELEGRAM_INTERFACE_NAME) {
    return telegram.sendPhoto(interface.chatId, photoPath);
  }

  throw new Error('Unknown interface name');
};

File: src/common/actions.py
import os
from src.common import telegram

_TELEGRAM_INTERFACE_NAME = os.environ.get('TELEGRAM_INTERFACE_NAME')

def reply_document(interface, document_path):
  if interface['name'] == _TELEGRAM_INTERFACE_NAME:
    return telegram.send_document(interface['chatId'], document_path)

  raise Exception('Unknown interface name')

def reply_message(interface, message, parse_mode=None):
  if interface['name'] == _TELEGRAM_INTERFACE_NAME:
    return telegram.send_message(interface['chatId'], message, parse_mode)

  raise Exception('Unknown interface name')
